.circleci/config.yml

```diff
@@ -758,6 +774,13 @@ workflows:
             tags:
               only: /^v.*$/
 
+      - upload_build_artifacts:
+          requires:
+            - linux_package
+          filters:
```

Comment:
Could you please remind me what this filter means?

Before this change it was only on master. Now it looks like it is on:
* Tags matching `/^v.*$/`
* **All** branches? (I am not sure what default for branches is, and whether default branches is documented in Circle CI doc - [I did not find it](https://circleci.com/docs/2.0/configuration-reference/#filters-1).)

Reply:
Yes, all branches. If there is no branch filters, it runs for all branches.
it's documented in the docs you refer:
> If neither only nor ignore are specified then all branches will run the job.

> Before this change it was only on master. Now it looks like it is on:

This PR should not change the behaviour. However, the build artefact are uploaded for each branch build and on all release tags. Also the job itself have an extra check that uploads the "latest" build artefact on master only.

Reply:
I see, I was reading incorrectly, it does not change semantics, correct.

----

I see this snippet is kept:
```
            aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
            if [ -n "$CIRCLE_BRANCH" -a "$CIRCLE_BRANCH" = "master" ]; then
              aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/epoch-latest-${PKG_SUFFIX:?}.tar.gz
            fi
```